openstack/loadbalancer/v2/flavorprofiles/doc.go

```diff
@@ -0,0 +1,57 @@
+/*
+Package flavorprofiles provides information and interaction
+with FlavorProfiles for the OpenStack Load-balancing service.
+
+Example to List FlavorProfiles
+
+	listOpts := flavorprofiles.ListOpts{}
+
+	allPages, err := flavorprofiles.List(octaviaClient, listOpts).AllPages()
+	if err != nil {
+		panic(err)
+	}
+
+	allFlavorProfiles, err := flavorprofiles.ExtractFlavorProfiles(allPages)
+	if err != nil {
+		panic(err)
+	}
+
+	for _, flavorProfile := range allFlavors {
+		fmt.Printf("%+v\n", flavorProfile)
+	}
+
+Example to Create a FlavorProfile
+
+	createOpts := flavorprofiles.CreateOpts{
+		Name:         "amphora-single",
+		ProviderName: "amphora",
+		FlavorData:   "{\"loadbalancer_topology\": \"SINGLE\"}",
+	}
+
+	flavorprofile, err := flavorprofiles.Create(octaviaClient, createOpts).Extract()
```

Comment:
nit -- try to use camel case for variable names:
```suggestion
	flavorProfile, err := flavorprofiles.Create(octaviaClient, createOpts).Extract()
```